### DISCUSSION TITLE
Homebridge or Home Assistant - what to pick ?

### DISCUSSION THREAD

**Discussion by simbaaman786** [community/community#145780] [Discussions]
### Select Topic Area

Question

### Body

Hello pal , As a person who is into smart home automation , always try finding ways to connect any devices on my phone , so i have more time to spend with myself than running a machine. I use Apple Homekit as my prime help ! It works great in home and remotely. I have not yet tried an alternative to connect any device that doesnt support homekit , but thinking of doing it now. I was stuck with two options to pick from - Homebridge or Home Assistant . obviously bridge has plugins that is just made for homekit , But is it easy to set up as this says ? This [articles ](https://connecthomekit.com/homebridge-vs-home-assistant/)does shows let me pick homebridge , But i am not friends with Github , Any suggestions is appreciated ! Please!

**Comment by Akash1134** [comment]
Hi @simbaaman786 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
